src/components/checkbox/Checkbox.css

```diff
@@ -0,0 +1,21 @@
+.Checkbox span:first-of-type {
+  background-color: #DDE6EB;
+  box-shadow: inset 0 1px 1px 0px rgba(0,0,0,0.18), 0 1px 0 0px rgba(255,255,255,0.75);
+}
+
+.Checkbox input {
```

Comment:
You can replace this with tachyons atoms

```jsx
<input className='absolute o-0'  type='checkbox' disabled={disabled} onChange={change} />
```

our css is always more specific than tachyons because they are all single classes and we're after it in the load order.